obpp3.json

```diff
@@ -949,7 +949,38 @@
                             "numeral": "1",
                             "id": "OBPPV3/CM31",
                             "description": "Avoid performing queries to third party services regarding specific transactions in a manner which reveals the IP address of the user",
-                            "effectiveness": 0
+                            "effectiveness": 0,
+                            "criteria-groups": [
+                                {
+                                    "criteria": [
+                                        {
+                                            "numeral": "a",
```

Comment:
Include new comment for CR24: "Tor, I2P, and VPNs are valid methods of hiding machine identity."